packages/components/src/slot-fill/bubbles-virtually/observable-map.ts

```diff
@@ -51,18 +51,25 @@ export function observableMap< K, V >(): ObservableMap< K, V > {
 			callListeners( name );
 		},
 		subscribe( name, listener ) {
-			let list = listeners.get( name );
-			if ( ! list ) {
-				list = new Set();
-				listeners.set( name, list );
-			}
+			const list = getListeners( name );
```

Comment:
Is this intended to be reused at some point?

Reply:
I extracted `getListeners` so that TypeScript stops bothering me 🙂 This code doesn't pass type check:
```js
let list = ...
if ( ! list ) {
  list = ...
}
return () => list.delete();
```
TypeScript (5.1.x) thinks that `list` can be `undefined` inside the body of the returned function. Because after the first line it really can be `undefined`. TypeScript 5.5 no longer has this bug. But it forced me to extract the `list`-assigning code to a function.